tests/abi/tools/revng-abi-verify/Verify.h

```diff
@@ -9,17 +9,21 @@
 enum ExitCode {
   Success = 0,
   FailedOpeningTheInputFile = 1,
-  FailedOpeningTheOutputFile = 2,
-  FailedOpeningTheArtifactFile = 3,
-  FailedDeserializingTheModel = 4,
-  FailedVerifyingTheModel = 5,
-  FailedArgumentCountCheck = 6,
-  FailedArgumentCompatibilityCheck = 7,
-  FailedLocatingAnArgument = 8,
-  FailedSelectingSingleArgumentLocation = 9
+  FailedOpeningTheArtifactFile = 2,
+  FailedDeserializingTheModel = 3,
+  FailedVerifyingTheModel = 4,
+
+  OnlyContinuousStackArgumentsAreSupported = 5,
+  UnknownRegister = 6,
+  UnknownStackOffset = 7,
+  ArgumentCouldNotBeLocated = 8,
+  FoundUnexpectedReturnValue = 9,
+  ExpectedReturnValueNotFound = 10,
+  UnknownReturnValueRegister = 11,
+  ReturnValueCouldNotBeLocated = 12,
+  CombinedStackArgumentsSizeIsWrong = 13
 };
```

Comment:
Please consider dropping this in favor of `llvm::createStringError`. In terms of error reporting, that's direction we want to go and I prefer to increase its adoption w.r.t. custom error handling.

Reply:
Sure, I'll take a look.

Reply:
What about something like this?
https://github.com/revng/revng/pull/148/commits/90621ae1874c78344be2e4cf838b72aaf15485c7

Reply:
I don't think there's much benefit in using `std::error_code`. Either it's something you can act on (and then you should probably use a custom error, not a string error) or let's just go with error string + `llvm::inconvertibleErrorCode`, which is what we do in other parts of the codebase.

Reply:
The whole point is that I want the code to be carried out of the test - it's easier for me to find the problem having "`revng-abi-verify` failed with code 11" rather than some log entry about what happened. It's probably better for other people to have the message - that's why both seems like a solid middle ground.